.github/workflows/semantic-pr.yml

```diff
@@ -0,0 +1,21 @@
+name: Check semantic PR titles
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+
+permissions:
+  contents: read
+
+jobs:
+  title-check:
+    runs-on: ubuntu-24.04
```

Comment:
use `ubuntu-latest` which is what every other thing checks.

but now I have a different idea -- can't we fold this into the pre-commit check we run by adding another step? why add a completely different workflow?

Reply:
Great suggestion!

Reply:
cool, could you just make it `ubuntu-latest`?

Reply:
Done!